AtData/AtFittedTrack.h

```diff
@@ -0,0 +1,144 @@
+#ifndef ATFITTEDTRACK_H
+#define ATFITTEDTRACK_H
+
+#include <Math/Point3D.h>
+#include <Math/Point3Dfwd.h>
+#include <Math/Vector3D.h>
+#include <Math/Vector3Dfwd.h>
+#include <Rtypes.h>
+#include <TMath.h>
+#include <TObject.h>
+
+#include <algorithm>
+#include <iostream>
+#include <memory>
+#include <utility>
+#include <vector>
+
+class TBuffer;
+class TClass;
+class TMemberInspector;
+
+class AtFittedTrack : public TObject {
```

Comment:
In the future I will look at changing this class to rely less on the tuples and generalize it to apply to both common fitting methods (MC and Genfit). Over the last FRIB experiment, I started this work but like we discussed it is not worth waiting for.